### ISSUE TITLE
assertAttribute and assertDataAttribute

### ISSUE THREAD

**Issue by troccoli** [laravel/dusk#747]
Before I dwell into submitting a PR, which I can do no problem, is there an appetite for an `assertAttribute()` method?

**Use case**

I have a button that doesn't have any text and therefore I use the `aria-label` attribute. In my test I want to make sure not only that the button is there but also that it has the `aria-label` attribute and that it is the correct label.

```php
$browser->assertAttribute('#button', 'aria-label', 'Add an item');
```

Additionally, I could add `assertDataAttribute()` and maybe even `assertAriaAttribute()` so I could write

```php
$browser->assertAriaAttribute('#button', 'label', 'Add an item');
```

What do you think?

**Comment by driesvints** [comment]
This looks interesting to me but the decision will be with Taylor in the end. Feel free to attempt a PR.

**Comment by troccoli** [comment]
#751

**Comment by u01jmg3** [comment]
I think this can be closed?
